build_tools/python_deploy/build_windows_packages.sh

```diff
@@ -69,19 +69,22 @@ function run() {
 
 function build_iree_runtime() {
   local python_version="$1"
+  IREE_HAL_DRIVER_CUDA=ON \
   IREE_HAL_DRIVER_VULKAN=ON \
   py -${python_version} -m pip wheel -v -w $output_dir $repo_root/runtime/
 }
 
 function build_iree_runtime_instrumented() {
   local python_version="$1"
+  IREE_HAL_DRIVER_CUDA=ON \
   IREE_HAL_DRIVER_VULKAN=ON IREE_ENABLE_RUNTIME_TRACING=ON \
   IREE_RUNTIME_CUSTOM_PACKAGE_SUFFIX="-instrumented" \
   py -${python_version} -m pip wheel -v -w $output_dir $repo_root/runtime/
 }
 
 function build_iree_compiler() {
   local python_version="$1"
+  IREE_TARGET_BACKEND_CUDA=ON \
   py -${python_version} -m pip wheel -v -w $output_dir $repo_root/compiler/
 }
```

Comment:
This shouldn't be needed, and we should try to match the other scripts

https://github.com/openxla/iree/blob/cb33f73d7d5e6c33d994316fa8659aed0b4b3534/build_tools/python_deploy/build_linux_packages.sh#L165-L168 https://github.com/openxla/iree/blob/cb33f73d7d5e6c33d994316fa8659aed0b4b3534/build_tools/python_deploy/build_macos_packages.sh#L95-L97

The CUDA backend defaults to enabled on platforms where it is "available" (either SDK already installed or running on a platform where we have a script to install automatically), as of https://github.com/openxla/iree/pull/11832 and https://github.com/openxla/iree/pull/11976

Reply:
ah ok. let me test to make sure it is enabled once I get a good build.  But using awk etc would be awk-ward on Windows :) so will hopefully avoid it.

Reply:
Huh... https://github.com/openxla/iree/actions/runs/5765425296/job/15631341317

```
  -- IREE HAL drivers:
  --   - cuda
  --   - local-sync
  --   - local-task
  --   - vulkan
  -- IREE compiler target backends:
  --   - llvm-cpu
  --   - llvm-cpu (wasm)
  --   - metal-spirv
  --   - vulkan-spirv
  --   - vmvx
```

do I have that inverted? HAL driver is already enabled, target backend isn't?

Reply:
Oh, we turn the CUDA HAL off unless it's explicitly requested, rather than rely on the default-enabled-if-available logic: https://github.com/openxla/iree/blob/cb33f73d7d5e6c33d994316fa8659aed0b4b3534/runtime/setup.py#L252

Reply:
same thing on the compiler side: https://github.com/openxla/iree/blob/cb33f73d7d5e6c33d994316fa8659aed0b4b3534/compiler/setup.py#L260

I think we should be able to remove both of those